frame/aura/src/lib.rs

```diff
@@ -162,6 +163,24 @@ impl<T: Trait> FindAuthor<u32> for Module<T> {
 	}
 }
 
+pub struct FindAccountFromAuthorIndex<T, Inner>(sp_std::marker::PhantomData<(T, Inner)>);
```

Comment:
```suggestion
/// We can not implement `FindAuthor` twice, because the compiler does not know if 
/// `u32 == T::AuthorityId` and thus, prevents us to implement the trait twice.
#[doc(hidden)]
pub struct FindAccountFromAuthorIndex<T, Inner>(sp_std::marker::PhantomData<(T, Inner)>);
```